esp32-rust/compile.sh

```diff
@@ -38,6 +40,6 @@ mkdir -p build-out
 if [ -f build-in/main.rs ]; then
   cat build-in/main.rs > examples/ledc-simple.rs
 fi
-
+pip3 install esptool
```

Comment:
Why not install it as part of the docker image? this way we don't have to spend time running pip3 whenever we compile a project

Reply:
How about adding the following at the top of the Dockerfile (right after the FROM command)?
I tried it and it seemed to do the trick:

```
# Install esptool
RUN IDF_PATH=/home/esp/.espressif/frameworks/esp-idf-v4.4 . "/home/esp/.espressif/frameworks/esp-idf-v4.4/export.sh" \
    && pip3 install esptool
```